src/main.rs

```diff
@@ -486,12 +486,10 @@ fn get_history_path() -> PathBuf {
     xdg_dirs.place_cache_file("repl_history.txt").unwrap()
 }
 
-fn get_prompt(vm: &VirtualMachine, prompt_name: &str) -> String {
```

Comment:
I would like to keep the old signature and return a `String` here. What is the benefit of returning the `Option` here?

Reply:
It doesn't clone the string this way, `vm.to_str` returns a `PyStringRef` whose inner `String` is already held inside of a PyObjectRef on the heap, but `vm.to_pystr` clones that `String` when all `repl.readline()` needs is an `&str`. I know it's a really tiny micro-optimization, especially when 99% of the time the string's only going to be 5 characters long, but I just like doing them.